charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,104 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports finability, reusability and reproducability with in the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who assists to data contributors wanting to upload data into the DCP
```

Comment:
s/ingest support team/**I**ngest **S**upport team/
s/assists to/assists/

Reply:
fixed